misk/src/main/kotlin/misk/hibernate/Session.kt

```diff
@@ -0,0 +1,11 @@
+package misk.hibernate
+
+import org.hibernate.query.Query
+import javax.persistence.criteria.CriteriaBuilder
+import javax.persistence.criteria.CriteriaQuery
+
+interface Session {
+  fun <T : DbEntity<T>> save(entity: T)
+  fun newCriteriaBuilder(): CriteriaBuilder
```

Comment:
is there anything Session specific about a `Criteria`? wondering if this belongs somewhere else